### ISSUE TITLE
Documentation of Email FromMobileMessage

### ISSUE THREAD

**Issue by brabiega** [afarran/swm2m_vms_tests#41]
It would be nice if the documentation of Email message mentioned when the message is sent.

Perhaps a short hint and a link to Mail module, where the logic of sending Email message is explained?

**Comment by afarran** [comment]
I agree. I think a link to VMS/doc/v1_4_1/modules/Mail.html should be added in: VMS/doc/v1_4_1/modules/FromMobileMessages.html#Email
